src/svm/svc.rs

```diff
@@ -108,7 +102,7 @@ pub struct SVCParameters<
     pub tol: TX,
     #[cfg_attr(feature = "serde", serde(skip_deserializing))]
     /// The kernel function.
-    pub kernel: Option<&'a dyn Kernel<'a>>,
+    pub kernel: Option<Box<dyn Kernel>>,
```

Comment:
this one is the same as writing `Option<&dyn Kernel>` as both are allocated at runtime, why change it>

Reply:
just to get ride of lifetimes associated to the type

Reply:
ok, but why using `Box`? `pub kernel: Option<&dyn Kernel>,` should do and it is more inline with Rust 2021

Reply:
I think that compiler will complain in  `Option<&dyn Kernel>,`, in that case I think that it is needed a explicit lifetime definition

Reply:
I will try that in any case. Could you share with me a reference to the Rust 2021 suggestion?

Reply:
https://doc.rust-lang.org/edition-guide/rust-2018/new-keywords.html

Reply:
I prefer to use `Option<&dyn ...>` because is easier to access: `variable.as_ref().unwrap()` (if you don't want to move) instead of `*variable.as_ref().unwrap()`. Are there easier ways to borrow the content of a box without moving it?

Reply:
I think that we can use `deref()`